[BUG] CI GHA builds of Amlogic and Rockchip fail on aarch64 build host

### Describe the bug
<!-- A clear description of what the bug is. -->
CI GHA builds of Amlogic and Rockchip fail on aarch64 build host

need to check noble Dockerfile / and or / mkimage (it was working previously - I think)
Ref:
- https://github.com/LibreELEC/LibreELEC.tv/pull/7927

### How to reproduce
Steps to reproduce the behavior:
1. Build Amlogic or Rockchip targets

### Information
 - LibreELEC Version: [e.g. 9.2.1] LE13
 - Hardware Platform: [e.g. RPi3] Amlogic and Rockchip (all)

### Log file
<!-- Add debug log files (https://libreelec.wiki/how_to/provide_logfile) that we can search for errors. -->
```
./scripts/image mkimage
**** This system lacks the following tools needed to build  ****
/usr/x86_64-linux-gnu/lib/ld-linux-x86-64.so.2 provided by libc6-amd64-cross
/usr/x86_64-linux-gnu/lib/libc.so.6 provided by libc6-amd64-cross
**** The system appears to be a ubuntu system ****
**** The following packages were not installed correctly ****
/usr/x86_64-linux-gnu/lib/ld-linux-x86-64.so.2 provided by libc6-amd64-cross
/usr/x86_64-linux-gnu/lib/libc.so.6 provided by libc6-amd64-cross
********
*********** FAILED COMMAND ***********
${SCRIPTS}/checkdeps
**************************************
make: *** [Makefile:10: image] Error 1
```

--- a/tools/docker/jammy/Dockerfile
+++ b/tools/docker/jammy/Dockerfile
@@ -29,13 +29,7 @@ RUN apt-get install -y \
     && ln -s /usr/lib/go-1.21/bin/gofmt /usr/bin/gofmt
 
 RUN if [ "$(uname -m)" = "aarch64" ]; then \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse' > /etc/apt/sources.list.d/amd64.list; \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted universe multiverse' >> /etc/apt/sources.list.d/amd64.list; \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse' >> /etc/apt/sources.list.d/amd64.list; \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-backports main restricted universe multiverse' >> /etc/apt/sources.list.d/amd64.list; \
-  apt-get update; \
-  dpkg --add-architecture amd64; \
-  apt-get install -y libc6:amd64 qemu-user-binfmt --no-install-recommends; \
+  apt-get install -y libc6-amd64-cross qemu-user-binfmt --no-install-recommends; \
  fi
 
 RUN rm -rf /var/lib/apt/lists/*

--- a/tools/docker/noble/Dockerfile
+++ b/tools/docker/noble/Dockerfile
@@ -25,13 +25,7 @@ RUN apt-get install -y \
     --no-install-recommends
 
 RUN if [ "$(uname -m)" = "aarch64" ]; then \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ noble main restricted universe multiverse' > /etc/apt/sources.list.d/amd64.list; \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ noble-updates main restricted universe multiverse' >> /etc/apt/sources.list.d/amd64.list; \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ noble-security main restricted universe multiverse' >> /etc/apt/sources.list.d/amd64.list; \
-  echo 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ noble-backports main restricted universe multiverse' >> /etc/apt/sources.list.d/amd64.list; \
-  apt-get update; \
-  dpkg --add-architecture amd64; \
-  apt-get install -y libc6:amd64 qemu-user-binfmt --no-install-recommends; \
+  apt-get install -y libc6-amd64-cross qemu-user-binfmt --no-install-recommends; \
  fi
 
 RUN rm -rf /var/lib/apt/lists/*

